README.md

```diff
@@ -1,78 +1,38 @@
 # DevTools RFCs
 
-Most changes to our tools, including bug fixes and documentation improvements, can be implemented and reviewed via the normal GitHub pull request workflow.
+Some changes to DevTools need feedback from the team and community to reach a consensus. In such a case you should use this repository to discuss the cahnge proposal proposal.
 
-Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the [DevTools core team][].
+DevTools RFC process aims to be lightweight and focused on conversations. Therefore opening a DevTools RFC is as easy as opening an issue in this repository to discuss the proposal.
 
-The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the framework.
+The [DevTools core team][] will be ultimately responsible for accepting or rejecting a change proposal, but the discussion is open to everybody.
```

Comment:
This link is wrong, it leads to the team working on the devtools-core repository. I think we created an RFC team, right? With a subset of the Mozilla DevTools team in it. We should point to that.

Reply:
So we have this team: https://github.com/orgs/devtools-html/teams/devtools-rfc/members

Need to remove Bryan and check with jryans if he still wants to participate.